src/MOI_wrapper.jl

```diff
@@ -118,397 +210,1291 @@ end
 
 MOI.get(::Optimizer, ::MOI.SolverName) = "Gurobi"
 
-function MOI.empty!(model::Optimizer)
-    MOI.empty!(model, model.env)
-    setparam!(model.inner, "InfUnbdInfo", 1)
-    for (name, value) in model.params
-        setparam!(model.inner, name, value)
+function MOI.supports_constraint(::Optimizer, F::Type{<:MOI.AbstractFunction}, S::Type{<:MOI.AbstractSet})
+    return (F, S) in SUPPORTED_CONSTRAINTS
```

Comment:
`MOI.supports_constraint(::Optimizer, ::Type{<:Union{...}}, ::Type{<:Union{...}}) = true` would be more efficient as Julia will be able to do constant propagation. The efficiency of `supports_constraint` is important for bridges